components/script/webdriver_handlers.rs

```diff
@@ -185,6 +180,44 @@ pub fn handle_get_title(page: &Rc<Page>, _pipeline: PipelineId, reply: IpcSender
     reply.send(String::from(page.document().Title())).unwrap();
 }
 
+pub fn handle_get_rect(page: &Rc<Page>,
+                       pipeline: PipelineId,
+                       element_id: String,
+                       reply: IpcSender<Result<Rect<f64>, ()>>) {
+    reply.send(match find_node_by_unique_id(&*page, pipeline, element_id) {
+        Some(elem) => {
+            // https://w3c.github.io/webdriver/webdriver-spec.html#dfn-calculate-the-absolute-position
+            match elem.downcast::<HTMLElement>() {
+                Some(html_elem) => {
+                    // Step 1
+                    let mut x: i32 = 0;
+                    let mut y: i32 = 0;
+
+                    let mut offset_parent = html_elem.GetOffsetParent();
+
+                    // Step 2
+                    while let Some(element) = offset_parent {
+                        offset_parent = match element.downcast::<HTMLElement>() {
+                            Some(elem) => {
+                                x += elem.OffsetLeft();
+                                y += elem.OffsetTop();
+                                elem.GetOffsetParent()
+                            },
+                            None => None
+                        };
+                    }
+                    // Step 3
+                    Ok(Rect::new(Point2D::new(x as f64, y as f64),
+                                 Size2D::new(html_elem.OffsetWidth() as f64,
+                                             html_elem.OffsetHeight() as f64)))
+                },
+                None => Err(())
+            }
+        },
+        None => Err(())
+    }).unwrap()
```

Comment:
This function doesn't look like it has a return value. Perhaps you're missing a semi-colon here?